src/components/routes/retrohunt/components/json.tsx

```diff
@@ -0,0 +1,68 @@
+import Editor, { loader } from '@monaco-editor/react';
```

Comment:
Why is this file called json.tsx when it's a monaco editor for yara rules?

Reply:
I added a single instance of monaco editor to be called and I removed the JSONEditor.